tests/compat.py

```diff
@@ -1,10 +1,12 @@
 from __future__ import annotations
 
+import sys
 
-try:
-    import zipp  # nopycln: import
-except ImportError:
-    import zipfile as zipp  # noqa: F401, TC002
+
+if sys.version_info < (3, 8):
```

Comment:
Based on https://github.com/python-poetry/poetry/blob/c8ae868472d9e6e4fc281adc08f2d14e374d35c8/pyproject.toml#L84

Took the opportunity to export as `zipfile` rather than `zipp` as well, since `zipfile` is the one in stdlib, and `zipp` will most likely go away when we will remove support for `3.7`.